<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hall</class>
 <widget class="QWidget" name="hall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hall</string>
  </property>
  <widget class="QLabel" name="kuang">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border:1px solid rgb(0, 0, 0);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stuButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学生</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="missionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>48</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>任务</string>
   </property>
  </widget>
  <widget class="QPushButton" name="placeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>164</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>场地</string>
   </property>
  </widget>
  <widget class="QPushButton" name="organButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>86</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>组织</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBut">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertBut">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equipButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设备</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>710</width>
     <height>440</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Query">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
